features/auth/signup/Signup.test.tsx

```diff
@@ -336,70 +288,38 @@ describe("Signup", () => {
   });
 
   it("displays the guidelines form when only it and feedback are pending", async () => {
-    const state: SignupFlowRes.AsObject = {
-      needBasic: false,
+    const state: SignupFlow = {
+      isCompleted: false,
       needAccount: false,
       needAcceptCommunityGuidelines: true,
       needFeedback: true,
-      needVerifyEmail: false,
       flowToken: "token",
     };
     window.localStorage.setItem("auth.flowState", JSON.stringify(state));
     render(<View />, { wrapper });
     expect(await screen.findByText("Guideline 1")).toBeVisible();
   });
 
-  it("displays the feedback form when feedback and email are pending", async () => {
-    const state: SignupFlowRes.AsObject = {
-      needBasic: false,
+  it("displays the feedback form when  feedback is pending", async () => {
+    const state: SignupFlow = {
+      isCompleted: false,
       needAccount: false,
       needAcceptCommunityGuidelines: false,
       needFeedback: true,
-      needVerifyEmail: true,
       flowToken: "token",
     };
     window.localStorage.setItem("auth.flowState", JSON.stringify(state));
     render(<View />, { wrapper });
     expect(screen.getByText(QUESTIONS_OPTIONAL)).toBeVisible();
   });
 
-  it("displays the feedback form when only feedback is pending", async () => {
-    const state: SignupFlowRes.AsObject = {
-      needBasic: false,
-      needAccount: false,
-      needAcceptCommunityGuidelines: false,
-      needFeedback: true,
-      needVerifyEmail: false,
-      flowToken: "token",
-    };
-    window.localStorage.setItem("auth.flowState", JSON.stringify(state));
-    render(<View />, { wrapper });
-    expect(screen.getByText(QUESTIONS_OPTIONAL)).toBeVisible();
-  });
-
-  it("displays the verify email message when email is pending", async () => {
-    const state: SignupFlowRes.AsObject = {
-      needBasic: false,
-      needAccount: false,
-      needAcceptCommunityGuidelines: false,
-      needFeedback: false,
-      needVerifyEmail: true,
-      flowToken: "token",
-    };
-    window.localStorage.setItem("auth.flowState", JSON.stringify(state));
-    render(<View />, { wrapper });
-    expect(screen.getByText(t("auth:sign_up_completed_prompt"))).toBeVisible();
-  });
```

Comment:
`"displays the feedback form when only feedback is pending"` is now covered by `"displays the feedback form when feedback is pending"`

`"displays the verify email message when email is pending"` can't be implemented at the moment - https://github.com/Couchers-org/backend-v2/pull/62#issuecomment-1312543104

Reply:
Does that mean we are a bit stuck on the frontend now then, since the user will need to activate their email separately before they finish the sign up process... but we have no way of telling them that right now unless they checked their email?

Reply:
Yes. I've added an issue for this on the backend-v2 repo.
https://github.com/Couchers-org/backend-v2/issues/68